node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
dont think we want to commit this.

Reply:
Agreed.  The stock .gitignore includes utils/node_modules, so I'd expect it to not grab utils/node_modules/.yarn-integrity, but it clearly did.  

For now, I've removed .yarn-integrity from the repo.  Perhaps someone with better .gitignore experience can debug this and fix the .gitignore?